repair_order_template/views/repair_order_template.xml

```diff
@@ -95,10 +95,10 @@
         </field>
     </record>
 
-    <record id="view_repair_order_template_tree" model="ir.ui.view">
+    <record id="view_repair_order_template_list" model="ir.ui.view">
```

Comment:
keep `view_repair_order_template_tree`:
> (...) keep XML-IDs that contain tree in its name for avoiding subsequent changes in dependent modules.
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0